src/services/messages.service.ts

```diff
@@ -12,11 +12,12 @@ interface IMessageService {
 
 class MessageService implements IMessageService {
   private msgCache: { [msgId: string]: MessageStatus } = {};
-  private knownMessageTypes: RequestType[] = ['EXTERNAL_KEY_PROOF_OF_OWNERSHIP_REQUEST', 'KEY_LINK_PROOF_OF_OWNERSHIP_REQUEST'];
+  private knownMessageTypes: RequestType[] = ['EXTERNAL_KEY_PROOF_OF_OWNERSHIP_REQUEST', 'KEY_LINK_PROOF_OF_OWNERSHIP_REQUEST', 'KEY_LINK_TX_SIGN_REQUEST'];
```

Comment:
I think it's better to have:
```suggestion
private deprecatedMessageTypes: RequestType[] = ['EXTERNAL_KEY_PROOF_OF_OWNERSHIP_REQUEST'];
private supportedMessageTypes: RequestType[] = ['KEY_LINK_PROOF_OF_OWNERSHIP_REQUEST', 'KEY_LINK_TX_SIGN_REQUEST'];
  private knownMessageTypes: RequestType[] = [].concat(deprecatedMessageTypes, supportedMessageTypes)
```

Or something of that sort